backend/internal/games/handler.go

```diff
@@ -40,13 +40,21 @@ func Handler(context context.Context, request events.APIGatewayProxyRequest) (ev
 		if err != nil {
 			return events.APIGatewayProxyResponse{
 				StatusCode: http.StatusInternalServerError,
+				Headers: map[string]string{
```

Comment:
I don't think you need these headers anymore no?

I think we only needed them for the proxy request service to handle the preflight requests

Can you remove these and any other ones not in PreflightService and see if they work?